yarn.lock

```diff
@@ -3575,6 +3575,14 @@ astral-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/astral-regex/-/astral-regex-2.0.0.tgz#483143c567aeed4785759c0865786dc77d7d2e31"
   integrity sha512-Z7tMw1ytTXt5jqMcOP+OQteU1VuNK9Y02uuJtKQ1Sv69jXQKKg5cibLwGJow8yzZP+eAc18EmLGPal0bp36rvQ==
 
```

Comment:
Are we ok with all these added dependencies?

Reply:
We might be able to remove some of these dependencies we don't use. For instance, I don't think this PR uses redis for anything like the example we based the code off of did.

Reply:
agree